apps/web/src/app/fleet-room.css

```
@@ -359,6 +359,328 @@
+  padding: 8px 10px;
+  max-height: 190px;
+  overflow: auto;
+  border-radius: 8px;
+  background: var(--pl-color-bg-inset);
+  font-family: var(--pl-font-mono, ui-monospace, "SF Mono", Menlo, monospace);
+  font-size: 11.5px;
+  line-height: 1.5;
+}
+.flr__diag-logline {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)